src/Rector/Return_/CommandConstantReturnCodeRector.php

```diff
@@ -82,17 +81,16 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if (! $this->nodeNameResolver->isName($node, 'execute')) {
+        if (! $classReflection->hasMethod('execute')) {
```

Comment:
This is wrong, hasMethod verify method exists, not check existing method that return parent of it